addOns/graaljs/graaljs.gradle.kts

```diff
@@ -18,6 +18,17 @@ zapAddOn {
         bundledLibs {
             libs.from(configurations.runtimeClasspath)
         }
+
+        dependencies {
+            addOns {
+                register("commonlib") {
+                    version.set(">=1.23.0")
+                }
+                register("scripts") {
+                    version.set(">=45.1.0")
```

Comment:
Sorry, didn't notice earlier, these versions now need to be bumped.

Reply:
No worries, I should have caught that too.